### ISSUE TITLE
Improve docs: A clean example with foundation

### ISSUE THREAD

**Issue by jfrolich** [jfrolich/smoothie#5]
Currently the template in README.md is very complex, would probably help to convert it to foundation.

**Comment by stephenmoloney** [comment]
agree, I am a bit busy for last few weeks and for another few weeks though...

**Comment by jfrolich** [comment]
No problem added the issue to remind myself :)

**Comment by stephenmoloney** [comment]
I should be able to do this. I will try to get to it eventually. I thought that the best way will be to create a simple example repository. 
The `README` is a bit cluttered to be honest.

**Comment by kevinwo** [comment]
Some additional explanation on dependencies would be helpful as well. I got stuck for a bit before I understood I needed to install yarn and get a minimum version of node.

e.g., on running mix smoothie.init

`error check-dependencies@0.12.3: The engine "node" is incompatible with this module. Expected version ">=4.4 <5 || >=6.3".`
